### DISCUSSION TITLE
Connection to rpc using proxy

### DISCUSSION THREAD

**Discussion by Exelery** [wevm/viem#1521] [Question]
Hi! Is it possible to add proxy in http transport? O maybe there are some solutions with custom transports?

**Comment by aarifkhamdi** [comment]
@jxom Are there any chances that the proxy functionality will be added to http transport? I would like to create PR with such an improvement. Could you guide me how to do it, please?

It's very useful if you need to imitate the behavior of users

**Reply by aarifkhamdi** [reply]
@jxom ping
